scripts/bin/update-packages

```diff
@@ -240,17 +249,28 @@ _fetch_and_cache_tags() {
 	)"
 
 	echo "INFO: Fetching non-GitHub packages"
-	local LATEST_TAGS='' LATEST_TAGS_GITHUB=''
+	local LATEST_TAGS=''
```

Comment:
Okay I assume we're no longer merging them at the parsing step then.

Reply:
Emmmmm... It completely erased graphql scan results...
We are still merging them at the parsing step.

Reply:
Right yep I remember now.
That was the initial problem.